### PR TITLE
All features

### PR DESCRIPTION
**Pull request by Danie12345**
Add list structure, includes:
- Container for the list
- Adding new item to the list
- Checkbox for selection (not functional)
- Styling
- Change text inside item
- Focus in and out of elements displays different options
- 'Delete selected' button

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```
Comment (SajjadAhmad14) [inline dist/index.html:1]: Please make sure there is no linter error in the PR.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Daniel, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and description. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: dist/index.html (hunk 1 above)
Comment (Danie12345) [reply dist/index.html:1]: Thank you @SajjadAhmad14. I have been debugging the Lighthouse linter, and still, it fails to show me the actual errors. I'll provide an image of a report on my machine, generated by Lighthouse. 
image
What can I do about this?

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Danie12345,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**